c10/cuda/CUDAStream.cpp

```diff
@@ -321,6 +334,43 @@ cudaStream_t CUDAStream::stream() const {
   return ptr->stream;
 }
 
+CUDAAssert* CUDAStream::assert_state() const {
+  auto ptr = CUDAStream_internals(*this);
+  AT_ASSERT(ptr);
```

Comment:
`TORCH_INTERNAL_ASSERT` here

Reply:
(`AT_ASSERT` is the old name but we had a lot of people misusing it to do error checking for conditions that users could trigger. `TORCH_INTERNAL_ASSERT` is the "if this ever is false, that means we the library authors fucked up")